anchor/anchorRecord.go

```diff
@@ -123,47 +162,50 @@ func UnmarshalAnchorRecord(data []byte) (*AnchorRecord, error) {
 	return ar, nil
 }
 
-func UnmarshalAndValidateAnchorRecord(data []byte, publicKeys []interfaces.Verifier) (*AnchorRecord, bool, error) {
-	if len(data) == 0 {
-		return nil, false, fmt.Errorf("Invalid data passed")
+// verifyAnchorAndSignature verifies the data and signature from the public keys - unexported
```

Comment:
same here, "unexported" seems unecessary,

Reply:
Same clarifying comment as above, fixed to say it could be exported in the future.

Reply:
I removed the export comment